src/pivotal-ui/components/grids/grids.scss

```diff
@@ -2264,3 +2264,525 @@ page](/containers.html).
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+</div>
+<div class="frow flex-justify-center-xs">
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">justify center</div>
+ </div>
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+</div>
+<div class="frow flex-justify-end-xs">
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">justify end</div>
+ </div>
+</div>
+
+<hr>
+
+<div class="frow flex-align-start-xs">
+ <div class="fitem-xs mbxl"><div class="bg-neutral-10 pal" style="height: 5em">Tall</div></div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">align start</div>
+ </div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+</div>
+<div class="frow flex-align-center-xs">
+ <div class="fitem-xs mbxl"><div class="bg-neutral-10 pal" style="height: 5em">Tall</div></div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">align center</div>
+ </div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+</div>
+<div class="frow flex-align-end-xs">
+ <div class="fitem-xs mbxl"><div class="bg-neutral-10 pal" style="height: 5em">Tall</div></div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">align end</div>
+ </div>
+</div>
+
+*/
+
+.frow {
+  box-sizing: border-box;
+  display: flex;
+  flex: 0 1 auto;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-left: -(floor($grid-gutter-width / 2));
+  margin-right: -(ceil($grid-gutter-width / 2));
+}
+
+.fcol {
+  box-sizing: border-box;
+  display: flex;
+  flex: 0 1 auto;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.frow.reverse {
+  flex-direction: row-reverse;
+}
+
+.fcol.reverse {
+  flex-direction: column-reverse;
+}
+
+@mixin flex-base {
+  box-sizing: border-box;
+  flex: 0 0 auto;
+  .frow>& { // Only horizontally do we make gutters
+    padding-left: floor($grid-gutter-width / 2);
+    padding-right: ceil($grid-gutter-width / 2);
+  }
+}
+
+.fitem-xs {
+  @include flex-base;
+  flex-grow: 1;
+  max-width: 100%;
+}
+
+@for $i from 1 through $grid-columns {
+ .fitem-xs-offset-#{$i} {
+   @include flex-base;
+   margin-left: (100% / $grid-columns * $i);
+ }
+ .fitem-xs-#{$i} {
+   @include flex-base;
+   flex-basis: (100% / $grid-columns * $i) !important;
+   max-width: (100% / $grid-columns * $i);
+ }
+ @media (max-width: $screen-sm-min) {
```

Comment:
The bootstrap grids work the other way around... xs applies to all screen widths, sm applies to every screen width except xs... it might be confusing to have bootstrap grids be mobile first and flex grids be desktop first.